data/src/main/java/com/stop/data/repository/NearPlaceRepositoryImpl.kt

```diff
@@ -0,0 +1,24 @@
+package com.stop.data.repository
+
+import com.stop.data.remote.source.nearplace.NearPlaceRemoteDataSource
+import com.stop.domain.model.nearplace.Place
+import com.stop.domain.repository.NearPlaceRepository
+import javax.inject.Inject
+
+internal class NearPlaceRepositoryImpl @Inject constructor(
+    private val nearPlaceRemoteDataSource: NearPlaceRemoteDataSource
+) : NearPlaceRepository {
+
+    override suspend fun getNearPlaceList(
+        version: Int,
+        searchKeyword: String,
+        centerLon: Float,
+        centerLat: Float,
+        appKey: String
+    ): List<Place> {
+        return nearPlaceRemoteDataSource.getNearPlaceList(
+            version, searchKeyword, centerLon, centerLat, appKey
+        ).map { it.toUseCaseModel() }
```

Comment:
```suggestion
        return nearPlaceRemoteDataSource.getNearPlaceList(
            version = version,
            searchKeyword = searchKeyword,
            centerLon = centerLon,
            centerLat = centerLat,
            appKey = appKey,
        ).map { it.toUseCaseModel() }
```


함수에 입력하는 파라미터가 많을 때 변수명을 명시해줄 경우, 리팩터링할 때 수정하기 편합니다.
개인적인 취향이라 적용하지 않으셔도 됩니다.